BridgeCareApp/BridgeCare/Models/InventoryModel.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System.Collections.Generic;
+
+namespace BridgeCare.Models
+{
+    public class InventoryModel
+    {
+        public InventoryModel()
+        {
+            InventoryItems = new List<InventoryItemModel>();
```

Comment:
I think this can be made a member variable and can be made static.

Reply:
ok. Static part moved to file penndotcrosswalk, one copy of that. This is now used like regular model, not static..multiple different copies can exist.